### DISCUSSION TITLE
Blob does not exist error

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2151] [Q&A] [answered]
I am getting the following errors.  The imodel seems to load OK but when I zoom in, these error start to appear( at least 30/35).  I was getting higher number of these errors when I loaded the model first time in the app, the frequency seems to have reduced.  In earlier loads, I was also getting a POST error after these GET error but seems that is not appearing anymore. 

I am using version 2.19.1

GET https://imodelbackendprodukssa01.blob.core.windows.net/411b9389-ed42-43f6-8d50-1c4025e45d57/tiles/19_1-0x3000000000b/ed21b0c4-e444-4079-bfcf-5182823bdf75_11a95a5ec3982ef7/-b-4-8-5-9-10?sv=2018-11-09&se=2021-08-28T23%3A59%3A59Z&sr=c&sp=rl&sig=EWJpQRBASSoMroDWQBPjHKWlpkVQk3vaJwWuhqXPK%2Bg%3D 404 (The specified blob does not exist.)  CloudStorage.ts:100 

requestResource			@	CloudStorage.ts:100
(anonymous)			@	CloudStorage.ts:73
async function (async)		
(anonymous)			@	CloudStorage.ts:67
retrieve			@	CloudStorage.ts:65
requestCachedTileContent	@	TileAdmin.ts:584
requestContent			@	TileRequestChannel.ts:297
dispatch			@	TileRequest.ts:82
dispatch			@	TileRequestChannel.ts:267
process				@	TileRequestChannel.ts:216
process				@	TileRequestChannel.ts:529
processQueue			@	TileAdmin.ts:693
process				@	TileAdmin.ts:359
eventLoop			@	IModelApp.ts:519
requestAnimationFrame (async)		
requestNextAnimation		@	IModelApp.ts:473
invalidateDecorations		@	Viewport.ts:306
invalidateScene			@	Viewport.ts:316
invalidateScene			@	Viewport.ts:2725
(anonymous)			@	TileAdmin.ts:271
(anonymous)			@	TileAdmin.ts:271
(anonymous)			@	TileAdmin.ts:671
raiseEvent			@	BeEvent.ts:98
onTileContentLoaded		@	TileAdmin.ts:641
setIsReady			@	Tile.ts:203
setGraphic			@	Tile.ts:292
setContent			@	Tile.ts:298
setContent			@	IModelTile.ts:132
handleResponse			@	TileRequest.ts:168
async function (async)		
handleResponse			@	TileRequest.ts:163
dispatch			@	TileRequest.ts:112
async function (async)		
dispatch			@	TileRequest.ts:82
dispatch			@	TileRequestChannel.ts:267
process				@	TileRequestChannel.ts:216
process				@	TileRequestChannel.ts:529
processQueue			@	TileAdmin.ts:693
process				@	TileAdmin.ts:359
eventLoop			@	IModelApp.ts:519
requestAnimationFrame (async)	*** THE ABOVE CALL SEQUENCE SEEMS TO REPEAT 15/16 TIMES ***

 *** CALL HEADER ****
Request URL: https://imodelbackendprodukssa01.blob.core.windows.net/411b9389-ed42-43f6-8d50-1c4025e45d57/tiles/19_1-0x3000000000b/ed21b0c4-e444-4079-bfcf-5182823bdf75_11a95a5ec3982ef7/-b-4-8-5-9-10?sv=2018-11-09&se=2021-08-28T23%3A59%3A59Z&sr=c&sp=rl&sig=EWJpQRBASSoMroDWQBPjHKWlpkVQk3vaJwWuhqXPK%2Bg%3D
Request Method: GET
Status Code: 404 The specified blob does not exist.
Remote Address: 52.239.187.36:443
Referrer Policy: strict-origin-when-cross-origin
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: x-ms-request-id,x-ms-version,x-ms-error-code
Content-Length: 215
Content-Type: application/xml
Date: Thu, 26 Aug 2021 17:22:24 GMT
Server: Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0
x-ms-error-code: BlobNotFound
x-ms-request-id: 7f9f2131-e01e-0035-669e-9a0610000000
x-ms-version: 2018-11-09
Accept: */*
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.9
Connection: keep-alive
Host: imodelbackendprodukssa01.blob.core.windows.net
Origin: http://localhost:3000
Referer: http://localhost:3000/
sec-ch-ua: "Chromium";v="92", " Not A;Brand";v="99", "Google Chrome";v="92"
sec-ch-ua-mobile: ?0
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: cross-site
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36
sv: 2018-11-09
se: 2021-08-28T23:59:59Z
sr: c
sp: rl
sig: EWJpQRBASSoMroDWQBPjHKWlpkVQk3vaJwWuhqXPK+g=

*****RESPONSE*****

<?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
RequestId:7f9f2131-e01e-0035-669e-9a0610000000
Time:2021-08-26T17:22:24.8604940Z</Message></Error>

**Comment by aruniverse** [comment, marked as answer]
This can be safely ignored, we dynamically generate and fetch tiles. On initial load we fetch all tiles necessary to display the view at a decent level of detail, but as you zoom in and the level of detail increases we need to request new tiles which haven't been created and cached yet. Eventually these should resolve. 

So basically this isn't an actual error.

**Reply by pmconne** [reply]
Yes, unfortunately browsers do not provide much control over the display of these "errors".
